src/library/baseplaylistfeature.cpp

```diff
@@ -473,7 +473,14 @@ void BasePlaylistFeature::slotExportPlaylist() {
     if (file_location.isNull() || file_location.isEmpty()) {
         return;
     }
+    // if the extension was not added add it manually
+    // https://bugreports.qt.io/browse/QTBUG-27186
     QFileInfo fileName(file_location);
+    if (fileName.suffix().isNull() || fileName.suffix().isEmpty()) {
```

Comment:
isEmpty() is sufficient

Reply:
yes.
and no: filenames containig dots would fall through, e.g. "AwesomeParty 21.11.2021"
so we need to check for the selected file type explicitely.

Reply:
QString::isNull() implies QString::isEmpty(), it is redundant

Reply:
I didn't question that.
> we need to check for the selected file type explicitely.

= we need to verify `fileName.suffix()` is indeed the selected file type, not part of the intended basename.